pint/registry.py

```diff
@@ -830,6 +830,8 @@ def _get_dimensionality_ratio(self, unit1, unit2):
             return 1
 
         dim1, dim2 = (self.get_dimensionality(unit) for unit in (unit1, unit2))
+        if dim1 == dim2:
+            return 1
         if not dim1 or not dim2 or dim1.keys() != dim2.keys():  # not comparable
```

Comment:
I'd make this a `elif` to help reading (even if there's no actual advantage because we `return` anyways)